.scalafmt.conf

```diff
@@ -0,0 +1,10 @@
+maxColumn = 100
+project.git = true
+project.excludeFilters = [ /sbt-test/, /input_sources/ ]
```

Comment:
we should exclude `/contraband-scala/` too otherwise scalafmt and contraband are going to keep fighting each other